deploy/test/test_cases/TEST_ID_28_secrets_rotation.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
This has a lot of duplication with test 27, could we make some common functions for each test to call?